android/sdl_android/src/androidTest/java/com/smartdevicelink/managers/video/VideoStreamManagerTests.java

```diff
@@ -21,6 +21,7 @@
 import com.smartdevicelink.proxy.rpc.OnHMIStatus;
 import com.smartdevicelink.proxy.rpc.OnTouchEvent;
 import com.smartdevicelink.proxy.rpc.RegisterAppInterfaceResponse;
+import com.smartdevicelink.proxy.rpc.SdlMsgVersion;
```

Comment:
@kostyaBoss Given that changes were made to VideoStreamManager.java, do changes need to be made to VideoStreamManagerTests.java?

Reply:
@santhanamk, I don't think so, we've made just formatting changes